worth2/protectivebehaviors/templatetags/quizsummary.py

```diff
@@ -28,20 +28,20 @@ def render(self, context):
         positive_answers = []
         for b in blocks:
             # assumption: each of these quiz types has one question
-            question = b.content_object.question_set.first()
-
             latest_submission = Submission.objects.filter(
```

Comment:
I'm looking at the NEPI tempatetag tests now - I'll do something similar for the PositiveQuizResponses tag.